tools/cli/src/args.rs

```diff
@@ -101,15 +101,20 @@ pub enum DeployCommand {
     },
 }
 
-#[derive(Clone, Debug, PartialEq, Default)]
+#[derive(Clone, Debug, PartialEq)]
 pub enum DeployTaskRequestor {
-    #[default]
     Deployer,
     Fixed(String),
-    Payment {
-        amount: u128,
-        denom: Option<String>,
-    },
+    Payment { amount: u128, denom: Option<String> },
+}
+
+impl Default for DeployTaskRequestor {
+    fn default() -> Self {
+        DeployTaskRequestor::Payment {
+            amount: 1_000_000,
+            denom: Some("uslay".to_string()),
```

Comment:
To get it from the config I need an access to the context, it would be more like
```rust

impl DeployTaskRequestor {
    // Initialize with the context
    pub fn new_with_ctx(ctx: &Context) -> Self {
        let chain_config = ctx.chain_config();
        DeployTaskRequestor::Payment {
            amount: 0,
            denom: Some(chain_config.denom.clone()), // Get the `denom` from chain_config
        }
    }
}
```
I will try it out now.

Reply:
Actually I just saw that the CLI is taking it automatically if provided `None`, so I will set it that way.